packages/xchain-radix/src/const.ts

```diff
@@ -0,0 +1,32 @@
+import { Network, RootDerivationPaths } from '@xchainjs/xchain-client/lib'
+import { Asset } from '@xchainjs/xchain-util/lib'
+
+/**
+ * Chain identifier for Radix.
+ * This constant represents the identifier for the Radix Chain.
+ */
+export const RadixChain = 'RADIX' as const
+
+export const MAINNET_GATEWAY_URL = 'https://mainnet.radixdlt.com'
+export const STOKENET_GATEWAY_URL = 'https://stokenet.radixdlt.com'
+
+export const XRD_DECIMAL = 18
+
+export const XrdAsset: Asset = {
```

Comment:
Can we please have a different `XrdAsset` for each network? This is because the resource address changes.

Reply:
we now have one for each network